open-metadata-resources/open-metadata-deployment/docker/atlas/README.md

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
This would fail our license check as we need a header such as
```
# SPDX-License-Identifier: Apache-2.0
# Copyright Contributors to the Egeria project.
```